runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -1653,3 +1653,67 @@ TR::S390VirtualUnresolvedReadOnlySnippet::getLength(int32_t estimatedSnippetStar
            + sizeof(int32_t); /*RIP offset to post snippet call in mainline*/
    }
 
+uint8_t *
+TR::S390InterfaceCallReadOnlySnippet::emitSnippetBody()
+   {
+   uint8_t *cursor = cg()->getBinaryBufferCursor();
+   getSnippetLabel()->setCodeLocation(cursor);
+   TR::Node *callNode = getNode();
+
+   TR::SymbolReference *interfaceCallReadOnlySymRef = cg()->symRefTab()->findOrCreateRuntimeHelper(TR_S390interfaceCallHelperMultiSlotsReadOnly,
+                                                                                                               false, false, false);
+   
+   TR::GlobalFunctionCallData dataDestination(interfaceCallReadOnlySymRef, callNode, cursor, cg(), self());
+
+   cursor = cg()->getObjFmt()->encodeGlobalFunctionCall(dataDestination);
+
+   uint8_t *helperCallRA = cursor;
```

Comment:
Why not just change the type of this variable to be `intptr_t` and remove the casts below?

Reply:
Fixed in https://github.com/eclipse/openj9/compare/ab82c5d254e0315b3c65722371fda75cc1d79a10..3282a01df297676d33999fca192d2613b5db1b09